build.gradle.kts

```diff
@@ -70,6 +73,14 @@ dependencies {
   implementation("net.bytebuddy:byte-buddy:1.15.3")
 }
 
+configurations {
+  implementation.configure {
+    exclude(module = "spring-boot-starter-web")
```

Comment:
why is this dependency required?

Reply:
It is not a requirement but an exclusion since we already import webflux counterpart